pyramid/config/util.py

```diff
@@ -152,7 +152,16 @@ def make(self, config, **kw):
                 weights.append(1 << n + 1)
                 preds.append(pred)
         if kw:
-            raise ConfigurationError('Unknown predicate values: %r' % (kw,))
+            from difflib import get_close_matches
```

Comment:
Comment for future reviewers, import statement inside the function so that we only have to import it when we are about to raise a `ConfigurationError`.